packages/effect/src/RateLimiter.ts

```diff
@@ -0,0 +1,18 @@
+import type { Effect } from "./Effect.js"
+import type { Duration } from "./index.js"
```

Comment:
```suggestion
import type { Duration } from "./Duration.js"
```

Reply:
Done. 

I assume this has already been considered, but I wonder if there's a lint rule which we can use to define these import rules.